tests/puppeteer/pages/FO/orderConfirmation.js

```diff
@@ -24,4 +25,12 @@ module.exports = class OrderConfirmation extends FOBasePage {
   isFinalSummaryVisible() {
     return this.elementVisible(this.orderSummaryContent, 2000);
   }
+
+  /**
+   * Check if checkbox of condition to approve is visible
+   * @returns {boolean}
+   */
+  isConditionToApproveCheckboxVisible() {
```

Comment:
I think the selector and the function must be in Checkout page not OrderConfirmation.

Reply:
Nice catch :) yes I will fix it